style.css

```diff
@@ -0,0 +1,38 @@
+body {
+    background-color: black;
```

Comment:
Doesn't do anything.  Unless you're worried about what the webpage might look like if the background image doesn't load?